ha/resource/alert_monitor_resource_agent.py

```diff
@@ -0,0 +1,378 @@
+#!/usr/bin/env python3
```

Comment:
Why this change in this task?

Reply:
Also, will this break R1?

Reply:
Here i needed one generic
class resource_agent.py ResourceAgent 
but which taken by HW/IEM RA

i just changed class hierarchy
from
```
ResourceAgent
IEMRA                         HWRA                                                   
```
to
```
                                           ResourceAgent
CortxServiceResourceAgent                                                    AlertMonitorResourceAgent
DynamicFidServiceRA                                                       IEMRA                         HWRA                                                   
```

Here i only renamed file not modified existing one.
It will not broke R1 as only rename happed but entry point resource agent name still same.

But R1 for main branch already broken as main branch created for JBOD setup only and not maintained and tested for R1. Which we can take separately.

Reply:
ok